bedrock/careers/templates/careers/benefits.html

```diff
@@ -6,32 +6,157 @@
-{% block page_desc %}At Mozilla, you can pursue your future while working to protect the future of the internet for everyone, everywhere.{% endblock %}
+{% block careers_content %}
+  <header class="c-careers-text-hero">
+    <div class="mzp-l-content c-text-hero-wrapper mzp-t-content-md">
+      <h1 class="c-text-hero-title">Benefits for real life</h1>
 
-{% block body_class %}benefits-page{% endblock %}
+      <div class="c-text-hero-body">
+        <p>We’re proud to offer our employees these outstanding benefits, which
+          set us apart from other companies in our industry.</p>
 
-{% block careers_content %}
-<section class="mzp-c-hero mzp-t-dark page-hero"></section>
-
-<section class="mzp-l-content">
-    <h2>Benefits</h2>
-    <p>
-        To ensure our employees and their families feel safe, cared for, and supported, we offer world-class benefits, including:
-    </p>
-    <ul class="mzp-u-list-styled">
-        <li>Flexible work environment (nearly half of Mozillians work remotely)</li>
-        <li>Industry-leading paid parental leave (up to 26 weeks of fully paid leave for childbearing parents and up to 12 weeks for non-childbearing parents)</li>
-        <li>Generous medical, dental and vision coverage for you and your family (including no payroll deductions in most countries)</li>
-        <li>Reimbursement for professional development (up to $3,000 USD/year)</li>
-        <li>A work setup including the latest hardware and software of your choice</li>
-        <li>Wellness programs—we reimburse up to $1700 USD/year for a variety of expenses like child and elder care, mental wellness, and personal enrichment</li>
-    </ul>
-</section>
-
-<div class="mzp-l-content">
-  <div class="centered">
-    <a class="mzp-c-button" href="{{ url('careers.listings') }}">Apply Now</a>
-  </div>
-</div>
-{% endblock %}
+        <p class="c-careers-button-wrapper">
+          <a href="{{ url('careers.listings') }}" class="mzp-c-button mzp-t-secondary" data-cta-type="button" data-cta-name="careers listings">See our open roles</a>
+        </p>
+      </div>
+    </div>
+  </header>
+
+  <section class="c-careers-benefits-intro c-careers-section">
+    <div class="mzp-l-content mzp-t-content-xl">
+      <div class="c-careers-section-intro">
+        <h2 class="c-careers-section-title">
+          What we offer
+        </h2>
+
+        <p>Our benefits package is world class. By offering a comprehensive,
+          industry-leading health and wellbeing program to our employees and their
+          families, we believe it’s easier for everyone to focus on the work we’re
+          here to do together. Here’s a sample of what’s included in our benefits
+          package. One thing to note: Global benefits vary by location, and a
+          recruiter can answer specific questions if you live outside the U.S.
+          Here’s a sampling of what we offer.</p>
+      </div>
+
+      <ul class="mzp-l-columns mzp-t-columns-three">
```

Comment:
So regarding this, I tried multiple different things in terms of alignment:
1. Instead of using `mzp-t-columns-three`, I used `mzp-t-columns-two`:
<img width="820" alt="Screen Shot 2022-11-23 at 11 31 44 AM" src="https://user-images.githubusercontent.com/42309026/203599193-0c4a4b22-c26b-4cae-8768-86c27019b7dd.png">



2. I played around with the grid layout and moved the last item to be more towards the right:
<img width="835" alt="Screen Shot 2022-11-23 at 11 32 34 AM" src="https://user-images.githubusercontent.com/42309026/203599404-4da40179-2c7c-4e14-8541-8d10d099d780.png">


3. I used `display: flex` instead of `display: grid` and played around with the widths of the list items:
<img width="820" alt="Screen Shot 2022-11-23 at 11 31 06 AM" src="https://user-images.githubusercontent.com/42309026/203599066-0fe5d901-f252-453a-a468-7fe4e2c6361b.png">


I didn't personally like any of the solutions I tried out and liked the original layout better. What are your thoughts?

Reply:
eh.. I don't think there is a good solution here. Thanks for checking it out!